transmit-lib/src/cmr/transmit/metadata_db.clj

```diff
@@ -162,17 +162,25 @@
 
 (defn find-latest-concept
   "Searches metadata db for the latest concept matching the given parameters. Do not throw serivce
-  excpetion, returns the status and error message in a map in case of error."
+  exception, returns the status and error message in a map in case of error."
```

Comment:
Looks like you just want to add a function called `find-latest-concepts`. It is a lot easier to do that with the existing `find-concepts` function (just addd `:latest true` to the params would do it) rather than dealing with this function.

Reply:
`find-concepts` does exactly what I need it to, using it will be much cleaner 👍